fw/http_sess.c

```diff
@@ -197,39 +135,15 @@ tfw_http_sticky_build_redirect(TfwHttpReq *req, StickyVal *sv, RedirMarkVal *mv,
 	if (WARN_ON_ONCE(!req->vhost))
 		return TFW_HTTP_SESS_FAILURE;
 
-	/*
-	 * TODO: #598 rate limit requests with invalid cookie value.
-	 * Non-challengeable requests also must be rate limited.
-	 */
```

Comment:
I think that max_misses, which is checked in frang after this PR is used instead of  maximum number/rate limit of wrong sticky cookie values in #598. I am no correct? For non-challengeable requests there can be two cases:
1. If this request doesn't contain valid cookie we increment max_misses, if max_misses is exceed the limit in frang we block client, otherwise we only drop this request.
2. If this request contains valid cookie we process it successfully.

Reply:
#598 task is about Cookie sessions (without JSCH) , e.g. see https://github.com/tempesta-tech/tempesta/issues/598#issuecomment-414756609 . Suppose a bot can set cookies (even Mirai did this), but we don't wan't switch on JSCH to not to worsen user experience. If a client reaches sessions rate limit, then we switch on JSCH for them.